### ISSUE TITLE
Record lucky loots

### ISSUE THREAD

**Issue by groupsky** [DevJackSmith/mh-hunt-helper#42]
It would be interesting to record and later get a statistic about relationship between luck and lucky loot drops.

**Comment by DevJackSmith** [comment]
what exactly are you calling lucky loot and how do i get it?

**Comment by groupsky** [comment]
Quote from the [news](https://www.mousehuntgame.com/newspost.php?news_post_id=299):

> ### Lucky Loot Drops
> Luck has always helped to both capture mice as well as slightly boost the chance of them dropping loot. However, assessing the effect of your trap's luck on loot has always been tricky.
>
> A clover icon has been added next to loot items in your Hunter's Journal when that loot drop was a direct result of your trap's luck. When you see a loot drop with a clover next to it in your journal, it means that's an item you would not have looted if your trap setup did not have luck. Of course, the higher your luck, the more often you'll see those clover icons show up!

TL;DR when you have higher luck it's more likely you'll get a lucky loot. It's designation is a clover icon.

**Comment by DevJackSmith** [comment]
Thanks for clarifying, don't know how i missed it. Will try to replicate it
and then merge your pr.

**Comment by groupsky** [comment]
Closed in cef014e646d308962b102fc3ade0ece3a7974759
